dashboard/views.py

```diff
@@ -1,4 +1,4 @@
-"""Define the functions that handle various requests by returnig a view"""
+ra"""Define the functions that handle various requests by returnig a view"""
```

Comment:
why there is "ra" before comments?

Reply:
Looks like a typo; in fact  @jr-sawaliram and I faced a similar problem once while deploying to the server, which we quickly fixed. @Abirami8799 your keyboard sometimes types random letters by itself right? Maybe that's how these got added.